src/Command/FtpCommand.php

```diff
@@ -136,7 +136,7 @@ public function supportedSiteCommands() : array
             return $response['message'];
         }
 
-        return array_map('ltrim', $response['body']);
+        return array_map('ltrim', $response['body'] ?? []);
```

Comment:
The second parameter of `array_map` function should be `array` type. Not `null`.

It should use `??` syntax to handle the `null` situation.

Reply:
Hmm, the `$response` array come from the `FtpCommand::raw` method which always gonna return an array, if this method fails an exception will be thrown and no value will be returned. what you think ?

Reply:
The `$response` is always array if the method call is successful. But the `$response['body']` will be `null` in some situations.

For example, the following output is about `$response` array before calling the `array_map` function:

```
/home/lee/php-ftp-client/src/Command/FtpCommand.php:138:
array(6) {
  'response' =>
  array(1) {
    [0] =>
    string(20) "214 CHMOD UMASK HELP"
  }
  'code' =>
  int(214)
  'message' =>
  string(17) " CHMOD UMASK HELP"
  'body' =>
  NULL
  'end-message' =>
  NULL
  'success' =>
  bool(true)
}
```

Reply:
Okey you're right, not all FTP replies have multiple lines to extract the body from.